README.md

```diff
@@ -0,0 +1,50 @@
+# Reyan
+ Reyan is a free modern open-source application based on the latest feature of the Android/IOS development world which'll let you recite the holy quran easier than ever!
+
+## Tools And Technolagies
+* programming language: [Kotlin](https://kotlinlang.org/docs/reference/android-overview.html)
+* Architecture: [MVI](http://hannesdorfmann.com/android/model-view-intent)
+* MultiThreading: [Coroutines](https://kotlinlang.org/docs/reference/coroutines/coroutines-guide.html)
+* Android DI: [Dagger2](https://dagger.dev/users-guide)
+* General networking: [ktor.http](https://ktor.io/clients/http-client/engines.html)
+* Android networking: [OkHttp engine](https://square.github.io/okhttp)
+* Testing: [JUnit](https://junit.org/junit5)
+* Misc: [ExoPlayer](https://exoplayer.dev)
+* [Kotlin multi-platform](https://kotlinlang.org/docs/reference/multiplatform.html)
+
+## About
+  Reyan means "**Gift of God**" and Reyan applicaton, represents the most valuable gift of God which is The Holy Quran.
+  
+  Our goal is to make, reading the holy quran easier than ever and encourage our user to read more.
+ 
+# Contributions
+* All tasks and goals are listed in Trello board: [Quran Trello board](https://trello.com/b/pSsteeb5/quran)
+* Bug fixes and PR requests are highly appreciated and you're more than welcome to send us your feedbacks
+
+
+## Installing & Running
+
+ Download the Reyan source code from https://github.com/islamversity/Reyan (git clone https://github.com/islamversity/Reyan.git)
+ 
+For running Reyan you will be needing 
+* Android Studio version 4.0 or higher
+* Android gradle plugin version 4.0.1
+* Gradle version 6.3
+
+ Gradle.properties, keystore.jks and google-services.json are included in [this link ](https://drive.google.com/drive/folders/1TxL9DUeLN9fKmLFl1bMNtaH-1kOZV3Um) which you'll need to request access to.

```

Comment:
Noooooooo!
these are private!
you can suggest them to create their own keystore and google-services.json files and put them in the right places!